routes/users-router.js

```diff
@@ -0,0 +1,14 @@
+const { getUsers } = require("../models/users");
+const usersRouter = require("express").Router();
+
+usersRouter
+    .route("/")
+    .get((req, res) => {
+        getUsers().then(users => {
+            res.status(200).send({ users })
+        });
```

Comment:
Would get a .catch on here just in case - you'll need to add `next` to the parameter list